ci-operator/jobs/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x-periodics.yaml

```
@@ -0,0 +1,76 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: 1.15.x
+    org: syndesisio
+    repo: syndesis-qe
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: fuse-online-ocp4.13-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.13-lp-interop-fuse-online-interop-aws
```

Also add the [gangway-api](https://cspi-qe.gitbook.io/ocp-ci-onboarding/tutorials/triggering_guide#gangway-api) support below job name,  in order to trigger this job outside of the cron if needed.
```suggestion
  name: periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.13-lp-interop-fuse-online-interop-aws
    prowjob_defaults:
    tenant_id: gangway-api
```